### DISCUSSION TITLE
Is fuzz testing fetaure removed from PactumJS?

### DISCUSSION THREAD

**Discussion by notfounnd** [pactumjs/pactum#216] [Q&A] [answered]
Hi guys.

I was checking PactumJS documentation and I geted that fuzz testing feature was removed from docs.

Checking pull requests history I found this commit that removed the especification file from repository.

https://github.com/pactumjs/pactumjs.github.io/pull/19/files#diff-07b585537cbba80615afd84d8369b25dd9998f57dfcbab5e53113b2b37fcad13

I didn't found any release note about this change on Pactum repository project.

This feature really was removed?

**Comment by ASaiAnudeep** [comment, marked as answer]
Hello @notfounnd,

Thank you for starting this discussion. Fuzz testing capabilities are still in experimental phase and not yet removed but I'm planning to support it as a plugin.

Added the docs - https://pactumjs.github.io/guides/fuzz-testing.html

**Reply by notfounnd** [reply]
@ASaiAnudeep amazing, thanks for your help.
